lib/rules/arrow-spaces.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @fileoverview Rule to require parens in arrow function arguments.
+ * @author Jxck
+ * @copyright 2015 Jxck. All rights reserved.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+    /**
+     * Merge default rules to given optional rules.
+     * @param {Object} options Optional rules
+     * @returns {Object} Merged rules
+     */
+    function mergeRule(options) {
+        // default
+        var rule = { before: true, after: true };
+
+        // merge args
+        if (options.length > 0) {
+            var option = options[0];
+
+            if (typeof option.before !== "undefined") {
```

Comment:
Why not just say 

```js
rule.before = option.before !== false;
rule.after = option.after !== false;
```

The above statements mean that till the time user doesn't say `false`, its always true.